### ISSUE TITLE
SOCK_SEQPACKET

### ISSUE THREAD

**Issue by Globik** [de-vri-es/node-unix-socketpair#1]
How about sock_seqpacket? Have you tried?

**Comment by de-vri-es** [comment]
Yes, I did try. Creating the sockets works fine, but sadly there is no API in node for working with seqpacket sockets. Even datagram sockets are badly supported as far as I could tell.

I expected node to have a general libc wrapper, which would be enough, but there doesn't seem to be one :(

**Comment by Globik** [comment]
It's a pitty. So I'm trying implement seqsocket client in nodejs C addon napi land.

**Comment by de-vri-es** [comment]
That would be awesome. I'd be interested in following the progress. I'll also gladly add the `SEQPACKET` constant here when there's a usable API for them.
